### PR TITLE
Fix first time used connection

### PR DESCRIPTION
**Pull request by mouadino**
A newly created connection will have the UNKONWN state so until the heartbeat
greenlet get a chance to run, we will assume that this connection is alive.

This will fix first requests to services instead of timeouting.

### CODE REVIEW

File: lymph/core/connection.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def on_send(self, msg):
         self.sent_message_count += 1
 
     def is_alive(self):
-        return self.status in (RESPONSIVE, IDLE)
+        return self.status in (RESPONSIVE, IDLE, UNKNOWN)
```
Comment (mamachanko) [inline lymph/core/connection.py]: How about only allowing `UNKOWN` to be considered alive if there was no heartbeat yet, i.e. `self.explicit_heartbeat_count` is 0?

Comment (mouadino) [inline lymph/core/connection.py]: it's the current state, UNKOWN is only set as initial value for the connection so as soon as heartbeat start the state change to something else.

Comment (mamachanko) [inline lymph/core/connection.py]:
``` python
def is_alive(self):
    if self.explicit_heartbeat_count == 0:
        return self.status in (RESPONSIVE, IDLE, UNKNOWN)
    return self.status in (RESPONSIVE, IDLE)
```

**Comment by emulbreh** [comment]
+1, good enough for now

**Comment by mouadino** [comment]
I give up ! too brittle code :(

**Pull request closed without merging**